asb-native/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Mon Nov 23 23:00:56 IST 2020
+version=1.0.0
+groupId=org.ballerinalang
+artifactId=asb-native
```

Comment:
It is generated by Maven in the build. We can ignore target folder.

Reply:
Resolved. Thank you